.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,58 @@
+name: pull-request
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+    - source/**
+    - clik/**
+    - modules/**
+    - examples/**
+    - cmake/**
+    - hal/**
+    - .github/actions/do_build_ock/**
+    - .github/actions/setup_ubuntu_build/**
+    - .github/workflows/run_pr_tests.yml
```

Comment:
This is not part of this worflow so should not be triggered.

Reply:
Removed